profiles/cirrus/inputs.tf

```diff
@@ -0,0 +1,45 @@
+variable "project_name" {
+  description = "Project Name"
+  type        = string
+  validation {
+    condition     = length(var.project_name) <= 10
```

Comment:
I think we have 8 for this and 7 for environment defined elsewhere?

Reply:
Fixed